src/Nethermind/Nethermind.AuRa.Test/Contract/TestContractBlockchain.cs

```diff
@@ -21,7 +21,7 @@ protected TestContractBlockchain()
             SealEngineType = Nethermind.Core.SealEngineType.AuRa;
         }
 
-        public static async Task<TTest> ForTest<TTest, TTestClass>(string? testSuffix = null) where TTest : TestContractBlockchain, new()
+        public static async Task<TTest> ForTest<TTest, TTestClass>(string testSuffix = null) where TTest : TestContractBlockchain, new()
```

Comment:
What is the purpose of this change? Quite sure most compilers/IDEs will show it as a warning. If really needed, using `null!` will get rid of the warning.